cli/lib/delay.ts

```diff
@@ -41,27 +41,50 @@ const getLastBlockRpc = async (network: string) => {
 };
 
 const getLastBlockTheGraph = async (network: string) => {
-  const data = await request(
-    `https://api.thegraph.com/subgraphs/name/requestnetwork/request-payments-${network}`,
-    query,
-  );
+  let data;
+  if (network === "mantle-testnet") {
```

Comment:
see comment in cli/lib/status.ts.
I think you can have a common config for the root URL.